pkg/cache/informer_cache.go

```diff
@@ -152,6 +153,25 @@ func (ic *informerCache) GetInformer(ctx context.Context, obj client.Object) (In
 	return i.Informer, err
 }
 
+func (ic *informerCache) getInformerForKind(ctx context.Context, gvk schema.GroupVersionKind, obj runtime.Object) (bool, *internal.Cache, error) {
+	var cache *internal.Cache
+	var started bool
+	var err error
+	switch ic.missPolicy {
+	case MissPolicyFail:
+		var ok bool
+		cache, started, ok = ic.Informers.Peek(gvk, obj)
+		if !ok {
+			err = &errors.ErrResourceNotCached{GVK: gvk}
```

Comment:
Just return here?

Reply:
Yeah, didn't know what was easier to read, one return / control flow or inline returns